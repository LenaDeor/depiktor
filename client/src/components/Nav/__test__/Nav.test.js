import React from 'react';
import ReactDOM from 'react-dom';
import Nav from './../Nav';

import "@testing-library/jest-dom/extend-expect";
import renderer from "react-test-renderer";


it ("Renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Nav></Nav>, div)
})

it ("matches snapshot", () => {
  const tree = renderer.create(<Nav ></Nav>).toJSON();
  expect(tree).toMatchSnapshot();
})
