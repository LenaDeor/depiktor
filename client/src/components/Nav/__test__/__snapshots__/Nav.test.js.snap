// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="nav"
>
  <div
    className="logo"
  >
    depiktor
  </div>
</div>
`;
